wg-reliability/sippy/deployment.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: sippy
+  name: sippy
+  namespace: wg-reliability-sippy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sippy
+  template:
+    metadata:
+      labels:
+        app: sippy
+    spec:
+      containers:
+        - image: quay.io/bparees/sippy:kube-latest
+          imagePullPolicy: Always
+          name: sippy
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          resources:
+            limits:
+              memory: 1500M
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          command:
+            - /tmp/src/sippy
+          args:
+            - --server
+            - --end-day
+            - "7"
+            - --local-data
+            - /data
+            - --dashboard
+            - kube-master=sig-release-master-blocking,sig-release-master-informing=
+            - --variant=kube
+          volumeMounts:
+            - mountPath: /data
+              name: data
+        - image: quay.io/bparees/sippy:kube-latest
+          imagePullPolicy: Always
+          name: fetchdata
+          resources:
+            limits:
+              memory: 500M
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          command:
+            - /tmp/src/scripts/fetchdata-kube.sh
```

Comment:
What is this? I didn't see that anywhere.
I think you can simply run:
./sippy --fetch-data /usr/local/google/home/wojtekt/sippy_data --dashboard=kube-master=sig-release-master-blocking=

Or I guess, you might wanted this to run periodically already.
Do we want a container? Or do we want a CronJob that will be doing this?

Reply:
Ahh - I guess with CronJob, we may have a problem with mounting for different pods...

So we probably want this to be a container within a pod.